app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/overpaymentsmultiple_v2/BasisForClaimController.scala

```diff
@@ -41,7 +41,7 @@ class BasisForClaimController @Inject() (
 
   // Allow actions only if the MRN and ACC14 declaration are in place, and the EORI has been verified.
   final override val actionPrecondition: Option[Validate[OverpaymentsMultipleJourney]] =
-    Some(hasMRNAndDisplayDeclaration & declarantOrImporterEoriMatchesUserOrHasBeenVerified)
```

Comment:
why this change?

Reply:
Because it doesn't exist in staging

Reply:
I am going to test it with Madhu before merging it

Reply:
Apologies removed this change because I misunderstood what Madhu meant